bin/xbps-query/xbps-query.1

```diff
@@ -277,6 +282,73 @@ expression wins.
 This expects an absolute path.
 This mode only works with repositories.
 .El
+.Sh FORMAT STRINGS
+Variables are package properties if not otherwise documented.
+See
+.Sx PROPERTIES
+section for a list of available properties.
+.Pp
+As example a format string like:
+.Bd -offset indent -literal
+{pkgname:<30} {installed_size!humanize :>10}\\n
+.Ed
+.Pp
+Would produce a list formatted like:
+.Bd -offset indent -literal
+libxbps                            304 KB
+xbps                               484 KB
+.Ed
+.Pp
+Format strings are parsed by the following EBNF:
+.Bd -literal
+<grammar>      ::= (prefix | "\\" (escape|[{}]) | substitution)*
+<prefix>       ::= [^\\{}]+   -- Literal text chunk.
+<escape>       ::= [abfnrtv0] -- POSIX-like espace character.
+
+<substitution> ::= "{" variable ["?" default] ["!" conversion] [":" format] "}"
+<variable>     ::= [a-zA-Z0-9_-]
+
+<default>      ::= ([-]?[0-9]+)                            -- default number.
+                 | "true" | "false"                        -- default boolean.
+                 | ('"' (("\\" (escape|'"')) | [^"])* '"')  -- default string.
+
+<conversion>   ::= humanize | strmode
+
+-- Convert inode status information into a symbolic string
+<strmode> ::= "strmode"
+
+-- Format a number into a human readable form, the default is:`humanize .8Ki`:
+<humanize>     ::= "humanize" [space] [decimal] [width] [scale] [i]
+  <space>      ::= " "          -- Put a space between number and the suffix.
+  <decimal>    ::= "."          -- If the final result is less than 10, display
+                                   it using one digit.
+  <width>      ::= [0-9]+       -- Width of the output.
+  <scale>      ::= multiplier   -- Minimum scale multiplier and optionally
+                   [multiplier] -- Maxium scale multiplier.
```

Comment:
```suggestion
                   [multiplier] -- Maximum scale multiplier.
```

Reply:
fixed it and a couple others in https://github.com/void-linux/xbps/pull/541/commits/bc43563ad7bde173662a7998f5159215b9bab114